src/Counter.php

```diff
@@ -25,31 +25,19 @@ public function __construct(ClientInterface $client, string $baseUrl)
      * @param string $namespace
      * @param string $name
      * @param LabelList $tagList
+     * @param string $help
      * @throws \GuzzleHttp\Exception\GuzzleException
      */
-    public function count(string $namespace, string $name, LabelList $tagList): void
+    public function count(string $namespace, string $name, LabelList $tagList = null, string $help = ''): void
     {
-        $tagStr = $this->createTagString($tagList);
+        $tagStr = $tagList !== null ? $tagList->__toString() : '';
```

Comment:
One small suggestion. So $tagList implement a __toString method you can simply do a `$tagStr = (string)$tagList;

Reply:
Nice and done :+1: